Users.cs

```diff
@@ -1 +1,184 @@
-﻿
\ No newline at end of file
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.Data.SqlClient;
+namespace Library
+{
+    public partial class Users : Form
+    {
+        public Users()
+        {
+            InitializeComponent();
+            populate();
+        }
+
+        private void pictureBox1_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void panel1_Paint(object sender, PaintEventArgs e)
+        {
+
+        }
+        SqlConnection Con = new SqlConnection(@"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=C:\Users\Margar\Documents\LibraryDatabase.mdf;Integrated Security=True;Connect Timeout=30");
+        private void button1_Click(object sender, EventArgs e)
+        {
+            if (UsrName.Text == "" || Phone.Text == "" || Address.Text == "" || Passwrd.Text == "")
```

Comment:
easier to understand and less writing
```suggestion
            if((UsrName.Text ?? Phone.Text ?? Address.Text ?? Passwrd.Text) == ""){ ;}
```